js/start.js

```diff
@@ -417,6 +422,29 @@ var loadProfile = function(landingRoute, onboarded) {
               });
 
               $('#sideBar').html(app.chatVw.render().el);
+              startUpLoadingModal.remove();
+
+              // setting up a loadingModal instance that can be used app-wide as necessary.
+              app.loadingModal = new LoadingModal({
+                dismissOnOverlayClick: false,
+                dismissOnEscPress: false,
+                showCloseButton: false
+              }).render();
+              app.loadingModal.__origRemove = app.loadingModal.remove;
+              app.loadingModal.remove = () => {
+                throw new Error('This instance of the loadingModal is globally shared ' +
+                  'and should not be removed. When you are done with it, please close it.');
+              };
+
+              // setting up a simpleMessageModal instance that can be used app-wide as necessary.
+              app.simpleMessageModal = new SimpleMessageModal({
+                removeOnClose: false
+              });
+              app.simpleMessageModal.__origRemove = app.simpleMessageModal.remove;
+              app.simpleMessageModal.remove = () => {
+                throw new Error('This instance of the simpleMessageModal is globally shared ' +
+                  'and should not be removed. When you are done with it, please close it.');
+              };              
 
               app.router = new router({userModel: user, userProfile: userProfile, socketView: newSocketView});
 
```

Comment:
When launchOnboarding creates a new onboardingModal, shouldn't it get 

showCloseButton: false,
dismissOnOverlayClick: false,
dismissOnEscPress: false

As options? Being able to close the onboarding modal (by accident or on purpose) creates a weird experience where the loading screen appears briefly then the onboarding modal reappears.

There should probably be a startUpLoadingModal.close(); in launchOnboarding too, otherwise it's hanging around behind the onboarding modal, and can be closed by pressing Esc, which causes the screen to get lighter (because its overlay lightens the screen).